src/gt4py/next/otf/compilation/build_systems/compiledb.py

```diff
@@ -331,6 +336,17 @@ def _cc_create_compiledb(
             .replace(name, "$NAME")
         )
 
+    build_script_path = path / "build.sh"
+    try:
+        build_script_path.chmod(0o755)
+    except Exception:
+        pass
+    with build_script_path.open("w") as build_script_pointer:
+        build_script_pointer.write("#!/bin/sh\n")
+        for entry in compile_db:
+            build_script_pointer.write(f"{entry['command']}\n")
```

Comment:
This is not working, it contains the non expanded parameters. Anyway the build command is already included in the log, the thing that's missing is the configure script (but not too many use-cases for that, I guess). Anyway, I opened a PR in your fork with a proposal.

Reply:
Fixed